src/App.tsx

```diff
@@ -15,36 +16,124 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  SORT_BY_ALPHABET = 'Sort alphabetically',
+  SORT_BY_LENGTH = 'Sort by length',
+}
+
+export interface Change {
+  sortBy: SortType | null;
+  isReversed: boolean;
+}
+
+const sortByAlphabet = (isReversed: boolean, good1: string, good2: string) =>
+  isReversed ? good2.localeCompare(good1) : good1.localeCompare(good2);
```

Comment:
The function name `sortByAlphabet` doesn't start with a verb that indicates an action. Consider renaming it to `sortGoodsByAlphabet` to better describe what the function actually does. [CODE STYLE]